crates/node/src/txvalidation/event.rs

```diff
@@ -0,0 +1,185 @@
+use crate::txvalidation::download_manager;
+use crate::txvalidation::EventProcessError;
+use crate::types::{
+    transaction::{TransactionError, TxReceive, TxValdiated},
+    Transaction,
+};
+use crate::Mempool;
+use futures::future::join_all;
+use futures_util::TryFutureExt;
+use gevulot_node::types::transaction::AclWhitelist;
+use std::fmt::Debug;
+use std::net::SocketAddr;
+use std::path::Path;
+use tokio::sync::mpsc::UnboundedSender;
+
+//event type.
+#[derive(Debug, Clone)]
+pub struct RcvTx;
```

Comment:
```suggestion
pub struct ReceivedTx;
```
In cases like this, I think explicit naming is better than shortening. Helps code readability :slightly_smiling_face: 

Double checking: The past tense is probably correct here? The transaction _has been_ received and then it needs to be processed, right?

Reply:
Yes I change it.